### DISCUSSION TITLE
제페토 머티리얼 / 쉐이더 사용 제한 여부

### DISCUSSION THREAD

**Discussion by strangestraw** [naverz/zepeto-studio-kor#180] [5. Q&A]
안녕하세요.
제페토에서 아래와 같은 쉐이더 사용이 가능한가요?
![image](https://user-images.githubusercontent.com/5335151/148486629-8f746c03-e4d1-4014-9005-1ac42f08b0d2.png)
머티리얼 / 쉐이더 사용에 제한이 있는 지 궁금합니다.
가이드 부탁드립니다.
감사합니다.

**Comment by x4cf** [comment]
기본적인 쉐이더 스크립트는 정상작동합니다. 보여주신 이미지는 포스트프로세싱이들어간 이미지로보이는데 유니티 포스트프로세싱기능은 지원하지 않습니다

**Reply by strangestraw** [reply]
답변 감사드립니다!
